index.html

```diff
@@ -5050,6 +5050,17 @@ <h3>Working with numbers</h3>
 	<div class="req" id="loc_numbers_shape_display">
 	<p class="advisement">When formatting numeric values for display, allow for culturally sensitive display, including the use of non-ASCII digits (digit shaping).</p>
 	</div>
+
+	<div class="req" id="loc_numbers_lists">
+	<p class="advisement">When defining lists and counters, allow the style of the counters to be customized to cover the needs of cultures around the world.</p>
```

Comment:
I think this needs more work (see my previous comment) And would suggest:

```suggestion
	<p class="advisement">When defining a feature that automatically labels items incrementally for display to the user (such as when creating a numbered list), allow for localized presentation of the labels as well as for various counting/listing systems or styles.</p>
	<p>Examples of this can be found in <cite>CSS Counter Styles</cite> [[css-counter-styles]] and especially the accompanying <cite>Ready-made Counter Styles</cite> [[predefined-counter-styles]].</p>
```

Reply:
Thank you! I agree this is more specific and actionable for the spec authors and implementers than my previous text.